packages/build/src/build/buildRefs/runTransformer.js

```diff
@@ -22,7 +22,12 @@ async function runTransformer({ context, parsedFile, refDef }) {
       context,
       filePath: refDef.transformer,
     });
-    return transformerFn(parsedFile, refDef.vars);
+    try {
+      return transformerFn(parsedFile, refDef.vars);
+    } catch (error) {
+      context.logger.error(`Error running transformer ${refDef.transformer} for ${refDef.path}.`);
```

Comment:
We do :
```js
  } catch (error) {
    throw new Error(
      `Error calling resolver "${refDef.resolver}" from "${referencedFrom}": ${error.message}`
    );
  }
```
in `runRefResolver`.

Maybe we should be consistent between the two. I prefer the message here with the quotes, but I am not sure if doing context.logger.error or just throwing results in a better looking error for the user